bin/nib-stats.py

```diff
@@ -0,0 +1,18 @@
+#!python
+# emacs: -*- mode: python-mode; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+#
+#   See COPYING file distributed along with the NiBabel package for the
+#   copyright and license terms.
+#
+### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+"""
+Compute image statistics
+"""
+
+from nibabel.cmdline.stats import main
+
+
+if __name__ == '__main__':
+    main()
```

Comment:
These files are actually out-of-date. Just add an entry to the entry points:

https://github.com/nipy/nibabel/blob/917afabdbad80dfaf226bd1899745d4f5110cc0c/setup.cfg#L71-L81

Reply:
Great hadn't seen that!